### DISCUSSION TITLE
参考文献行间距问题

### DISCUSSION THREAD

**Discussion by menglinghui6** [obster-y/XJTU-thesis#68] [Q&A]
参考文献行距被老师指出和word版本不一致，想问一下怎么修改？

**Comment by obster-y** [comment]
不明白，请举例

在我看来是一致的

左边是 Word 生成的 pdf，右边是 LaTeX 生成的 pdf；绿线是 Word 一行的顶部，红线是 TeX 一行的顶部，因为 pdf 阅读器难以精准对齐，所以通过红线绿线体现；两条线之间的差异，在三行是基本一致的，因此很难说不一样

![ref](https://user-images.githubusercontent.com/55270856/170411843-b03c72b7-3fff-4e1e-b446-bf252c8d82de.png)

**Reply by Gausx** [reply]
> 不明白，请举例
> 
> 在我看来是一致的
> 
> 左边是 Word 生成的 pdf，右边是 LaTeX 生成的 pdf；绿线是 Word 一行的顶部，红线是 TeX 一行的顶部，因为 pdf 阅读器难以精准对齐，所以通过红线绿线体现；两条线之间的差异，在三行是基本一致的，因此很难说不一样
> 
> ![ref](https://user-images.githubusercontent.com/55270856/170411843-b03c72b7-3fff-4e1e-b446-bf252c8d82de.png)

![image](https://user-images.githubusercontent.com/55922272/170412958-b99bc513-db17-47f9-a51a-2a658369f014.png)
大概这个样子，间距不同。

尝试将致谢的间距，如图将1.5->1.3 对比起来和word 更加接近。所以想尝试将参考文献也设置为1.3，但是没有找到设置的位置
![image](https://user-images.githubusercontent.com/55922272/170413368-13aa3bda-12cc-44a0-a117-50bb589da4ba.png)

**Reply by obster-y** [reply]
致谢也没有这个啊，你确定你用的 dev 分支？
https://github.com/obster-y/XJTU-thesis/blob/7342f4bd78e9aa107330ca58da53bd3b13667e8d/LaTeX_Template/XJTU-thesis.cls#L1388-L1398

**Reply by Gausx** [reply]
> 致谢也没有这个啊，你确定你用的 dev 分支？
> 
> https://github.com/obster-y/XJTU-thesis/blob/7342f4bd78e9aa107330ca58da53bd3b13667e8d/LaTeX_Template/XJTU-thesis.cls#L1388-L1398

这个是我弄错了 ，这几句话是我自己添加的

**Reply by obster-y** [reply]
如果想要调整条目之间的间距 `\setlength{\bibitemsep}{0ex}` @Gausx

**Reply by Gausx** [reply]
> 如果想要调整条目之间的间距 `\setlength{\bibitemsep}{0ex}` @Gausx

收到谢谢答复

**Reply by Gausx** [reply]
> 如果想要调整条目之间的间距 `\setlength{\bibitemsep}{0ex}` @Gausx

您好，这个好像只是两个文献间距的距离调整，单个文献行内间距应该如何调整呢？

**Reply by obster-y** [reply]
biblatex 无法调整这一要求，需要直接调整行间距，建议通过设置字号解决

**Reply by Gausx** [reply]
> biblatex 无法调整这一要求，需要直接调整行间距，建议通过设置字号解决

好的谢谢
